java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -706,6 +741,7 @@ void copyHostBufferBytes(byte[] dst, int dstOffset, BufferType src, long srcOffs
    * @return - Boolean vector
    */
   public ColumnVector isNotNull() {
+    // No prediction possible because built up from other operators.
```

Comment:
I'm not following the comment here.  We know how many rows and thus approximately how big the resulting vector would be.  Is this trying to say the accounting is already handled within `validityAsBooleanVector` or...?

Reply:
Still wondering if this is saying it is impossible to make a prediction here (not sure why that would be the case?) or if this is saying the prediction is already being handled by the method being called.